### DISCUSSION TITLE
Suggestion: Redirect after verifying email + resetting password

### DISCUSSION THREAD

**Discussion by popovidis** [pocketbase/pocketbase#4961] [Q&A]
Hi there,

This would make the process of doing those actions very fluid rather than closing tab and then seeing the email.

Maybe add the url you wish to redirect to somewhere by the email settings.

I absolutely love this project and I am very happy it exists. It is awesome!

Thank you @ganigeorgiev

**Comment by ganigeorgiev** [comment]
You can already provide your own confirmation url and custom page by changing the "Action URL" in _Admin UI > Settings > Mail settings > {Expand template}_.

**Reply by popovidis** [reply]
I see. I meant more like leaving the current template as is and then instead of having close "Close" you could have a custom button which redirects to the specified url.
